tests/gocase/util/random.go

```diff
@@ -37,7 +37,7 @@ func RandPathNoResult(funcs ...func()) {
 
 // Random signed integer between -max and max (both extremes excluded).
```

Comment:
```suggestion
// Random signed integer in (-max, max)
```

Math has the sign for exclusive interval.